src/routes/stacks/+page.server.ts

```
@@ -22,12 +16,12 @@ export async function load() {
 		return {
 			stacks: combinedStacks
 		};
-	} catch (err) {
+	} catch (err: unknown) {
 		console.error('Failed to load stacks:', err);
 		const errorMessage = err instanceof Error ? err.message : String(err);
 		return {
 			stacks: [],
 			error: 'Failed to load Docker Compose stacks: ' + errorMessage
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)